sftp/sftp.go

```
@@ -119,7 +119,7 @@ func NewFs(name, root string) (fs.Fs, error) {
 	if sshClient, err := ssh.Dial("tcp", host + ":" + port, config); err != nil {
 		return nil,err
 	} else if sftpClient, err := sftp.NewClient(sshClient); err != nil {
-		sshClient.Close()
+		_ = sshClient.Close()
```

Don't ignore the error please!